modules/dxp/apps/osb/osb-testray/osb-testray-service/src/main/java/com/liferay/osb/testray/service/impl/CompareRunsLocalServiceImpl.java

```diff
@@ -14,18 +14,174 @@
 
 package com.liferay.osb.testray.service.impl;
 
+import com.liferay.object.model.ObjectDefinition;
+import com.liferay.object.petra.sql.dsl.DynamicObjectDefinitionTable;
+import com.liferay.object.service.ObjectDefinitionLocalService;
+import com.liferay.object.service.ObjectEntryLocalService;
+import com.liferay.object.service.ObjectFieldLocalService;
 import com.liferay.osb.testray.service.base.CompareRunsLocalServiceBaseImpl;
+import com.liferay.petra.sql.dsl.Column;
+import com.liferay.petra.sql.dsl.DSLQueryFactoryUtil;
+import com.liferay.petra.sql.dsl.Table;
+import com.liferay.petra.sql.dsl.query.JoinStep;
 import com.liferay.portal.aop.AopService;
+import com.liferay.portal.kernel.dao.orm.Criterion;
+import com.liferay.portal.kernel.dao.orm.DynamicQuery;
+import com.liferay.portal.kernel.dao.orm.RestrictionsFactoryUtil;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
 
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author José Abelenda
+ * @author Felipe Veloso
  */
 @Component(
 	property = "model.class.name=com.liferay.osb.testray.model.CompareRuns",
 	service = AopService.class
 )
 public class CompareRunsLocalServiceImpl
 	extends CompareRunsLocalServiceBaseImpl {
+
+	public int getComparison(
+		long companyId, long runIdA, long runIdB, String statusA,
```

Comment:
rename to testrayRunId1 and testrayRunId2, testrayDueStatus1 and testrayDueStatus2